source/connection-monitoring-and-pooling/tests/README.rst

```diff
@@ -0,0 +1,181 @@
+.. role:: javascript(code)
+  :language: javascript
+
+========================================
+Connection Monitoring and Pooling (CMAP)
+========================================
+
+.. contents::
+
+--------
+
+Introduction
+============
+
+The YAML and JSON files in this directory are platform-independent tests that
+drivers can use to prove their conformance to the Connection Monitoring and Pooling (CPAM) Spec.
+
+Several prose tests, which are not easily expressed in YAML, are also presented
+in this file. Those tests will need to be manually implemented by each driver.
+
+Common Test Format
+==================
+
+Each YAML file has the following keys:
+
+- ``version``: A version number indicating the expected format of the spec tests (current version = 1)
+- ``style``: A string indicating what style of tests this file contains. Currently ``unit`` is the only valid value
+- ``tests``: An array of tests that are to be run independently of each other
+
+Unit Test Format:
+=================
+
+All Unit Tests have some of the following fields:
+
+- ``description``: A text description of what the test is meant to assert
+- ``operations``: A list of operations to perform. Each operation has some of the following fields:
+
+  - ``command``: A string describing which command to issue.
```

Comment:
Naming this something other than "command" might be a good idea given that "command" usually refers to a server command; maybe "name" (or "operationName", if specificity is preferred)?

Reply:
I think I'll go with `name` since that is what the transactions tests use